python/sdk/merlin/pyfunc.py

```diff
@@ -369,3 +393,209 @@ def infer(self, model_input: pandas.DataFrame) -> Union[numpy.ndarray,
 
         """
         raise NotImplementedError("infer is not implemented")
+
+
+def run_pyfunc_model(
```

Comment:
would prefer this to be `run_local_pyfunc` or `run_pyfunc_local_model` to be more precise